allennlp/tests/commands/docstring_help_test.py

```diff
@@ -42,20 +42,16 @@ def test_docstring_help(self):
                 subcommand = match.group(2)
                 actual_output = _subcommand_help_output(subcommand)
 
-                self.assertEqual(
-                    expected_output,
-                    actual_output,
+                assert expected_output == actual_output, (
```

Comment:
Will this give me those pretty comparisons, that show immediately where the difference is, even in large data?

Reply:
Yes sir!